### ISSUE TITLE
`tox` should not exclude all of `koku/masu/test`

### ISSUE THREAD

**Issue by infinitewarp** [project-koku/koku#1205]
## User Story
As a developer, I want all of my code to be checked so I have high confidence in its quality. We should enforce standards consistently across our project code.

In `tox.ini`, all of the `koku/masu/test` file tree is excluded from the lint/flake8 environment. If we have problematic code, we should fix it, not allow it to remain (_and grow!_) ignored.

## Assumptions and Questions
- Can we fine-tune specific checks instead of excluding the whole tree?
- If there are specific conditions that we want to unilaterally ignore, we should configure for those.
  - …but only after raising to the group and agreeing.
- Implementation idea? Try `black`ing the code as a first pass...
- We are putting a soft cap on the effort involved here. If we cannot in reasonable time (a few days?) fix the issues, start the conversation with the team, and consider closing this as a "won't fix" **as a last resort**.

## Acceptance Criteria

- [x] Large sections of our code are not ignored by lint/flake8 checks
- [x] Someone has had a conversation with the team about reducing the number of linters and the general use/abuse of `# noqa` exceptions. This conversation may or may not result in a new issue; do not expect to completely resolve this debate as part of this issue.

**Comment by werwty** [comment]
Had discussion with team about removing pylint, resulted in https://github.com/project-koku/koku/issues/1471

**Comment by katherine-black** [comment]
Looks good, thank you so much!
